react-frontend/src/createRecognition/CreateRecognitionCard.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Card } from "react-bootstrap";
+
+import CreateRecognitionCardHeader from "./CreateRecognitionCardHeader";
+import CreateRecognitionCardBody from "./CreateRecognitionCardBody";
```

Comment:
we are having absolute imports supported `import CreateRecognitionCardBody from "CreateRecognitionCardBody";` should work, update other entries too